internal/ingestion/component/globals/globals_test.go

```
@@ -0,0 +1,85 @@
+
+package globals
+
+import (
+	"context"
+	"testing"
+
+	"ragflow/internal/agent/runtime"
+)
+
+// withChunkCap attaches a CanvasState carrying debug_chunk_cap to ctx.
+func withChunkCap(t *testing.T, cap int) context.Context {
```

golangci-lint (2.12.2)

[error] func withChunkCap is unused

(unused)